ReactWindows/ReactNative.Shared/Common/AccessibilityHelper.cs

```diff
@@ -0,0 +1,229 @@
+using ReactNative.UIManager;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+
+namespace ReactNative.Common
+{
+    /// <summary>
+    /// Accessibility helper methods.
+    /// </summary>
+    public static class AccessibilityHelper
+    {
+        #region ImportantForAccessibility
+
+        /// <summary>
+        /// Sets the ImportantForAccessibility property for uiElement.
```

Comment:
>uiElement [](start = 60, length = 9)

you can use <paramref> to avoid the xmldoc becoming stale if parameter names change.

Reply:
Fixed.

---
In reply to: [163125379](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r163125379) [](ancestors = 163125379)